### PR TITLE
netsniff-ng: 0.6.7 -> 0.6.8

### PR DESCRIPTION
**Pull request by fabaff**
###### Motivation for this change
Update to latest upstream release 0.6.8.

Change log: https://github.com/netsniff-ng/netsniff-ng/releases/tag/v0.6.8

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 109049` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>netsniff-ng</li>
  </ul>
</details>

File: pkgs/tools/networking/netsniff-ng/default.nix [hunk 1]
```diff
@@ -1,23 +1,53 @@
...
-    sha256 = "1jvihq30cwlpjqwny0lcrciysn40wscq6xik3s9b81nw2s7wiyqr";
+    sha256 = "10ih8amaqspy0zwg7hqvypa1v7ixpjl0n608cyfgyfzffp73lbqf";
   };
 
-  nativeBuildInputs = [ pkgconfig makeWrapper bison flex ];
+  nativeBuildInputs = [
+    bison
+    flex
+    makeWrapper
+    pkgconfig
```
Comment (SuperSandro2000) [inline pkgs/tools/networking/netsniff-ng/default.nix:36]:
```suggestion
    pkg-config
```

File: pkgs/tools/networking/netsniff-ng/default.nix [hunk 2]
```diff
@@ -1,23 +1,53 @@
...
+, geolite-legacy
+, libcli
+, libnet
+, libnetfilter_conntrack
+, libnl
+, libpcap
+, libsodium
+, liburcu
+, ncurses
+, pkgconfig
```
Comment (SuperSandro2000) [inline pkgs/tools/networking/netsniff-ng/default.nix:17]:
```suggestion
, pkg-config
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review which does not build all packages (e.g. lumo, tensorflow or pytorch)
If you find some bugs or got suggestions for further things to search or run please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 109049` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
1 package built:
</summary>
<ul>
<li>netsniff-ng</li>
</ul>
</details>

File: pkgs/tools/networking/netsniff-ng/default.nix (hunk 1 above)
Comment (fabaff) [reply pkgs/tools/networking/netsniff-ng/default.nix]: Thanks, updated.

File: pkgs/tools/networking/netsniff-ng/default.nix (hunk 2 above)
Comment (fabaff) [reply pkgs/tools/networking/netsniff-ng/default.nix]: Updated

**Pull request merged**